mapID_to_collection_DB/__init__.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+from pathlib import Path
+
+try:
+    import requests
+
+except ImportError:
+    raise ImportError("'Requests' module not found (Try running 'pip install requests')")
+
+try:
+    Path("settings.json").resolve()
+
+except (OSError, RuntimeError):
+    with open("settings.json", 'w') as f:
```

Comment:
This doesn't appear to actually create a `settings.json`. Even without a `settings.json` line 12 will not error out. `__init__.py` is also not called during `setup.py` or when `settings.json` is found to be missing where it likely should be.